jvm-libs/linea/clients/linea-l1-contract-client/src/main/kotlin/linea/contract/Web3JContractAsyncHelper.kt

```diff
@@ -289,12 +293,8 @@ class Web3JContractAsyncHelper(
     gasPriceCaps: GasPriceCaps?
   ): SafeFuture<String> {
     require(blobs.size in 0..6) { "Blobs size=${blobs.size} must be between 0 and 6." }
```

Comment:
A note for later: at some point in the future post EVM upgrade on L1, the 6 will need to be changed to be more than 6